Light_Install.rst

```diff
@@ -0,0 +1,68 @@
+**लाइट इनस्टॉल गाइड**
+
+मैटरमोस्ट एक मैसेजिंग और कोलैबोरेशन मंच है। मैटरमोस्ट के द्वारा,आप प्रतिदिन इस्तेमाल किए जाने वाले उपकरण एक ही जगह पर एकीकृत कर सकते है और कोई भी  अधिसूचना या कार्य नहीं छूटेगा।
+
+आरंभ करने से पहले आपको इनकी आवश्यकता होगी:
+- रूट लेवल एक्सेस वाला उबंटू सर्वर (18.04 या 20.04)
+- (रेकमेंडेड) आपके सर्वर से जुड़ा एक डोमेन नाम (यानी mymattermostserver.com)
+- (रेकमेंडेड) अपनी टीम के सदस्यों के ईमेल ताकि आप उन्हें सर्वर पर आमंत्रित कर सकें
+
+
+ **इंस्टालेशन**
+ -----------
+ 
+[मैटरमोस्ट ओमनीबस](https://docs.mattermost.com/install/mattermost-omnibus.html)  के द्वारा मैटरमोस्ट इनस्टॉल  करें।
+
+टर्मिनल विंडो खोलें और एंटेर करें:
+``curl -o- https://deb.packages.mattermost.com/repo-setup.sh | sudo bash``
+
+फिर,मैटरमोस्ट ओमनीबस इनस्टॉल करने  लिए ये चलाएं:
+``sudo apt install mattermost-omnibus``
+
+
+!नोट 
+यद्यपि ओमनीबस को इनस्टॉल करने और कॉन्फ़िगर करने का अनुशंसित तरीका SSL इनेबल्ड है, यदि आपको उसके बिना (या डोमेन नाम के बिना) उपयोग या परीक्षण करना चाहते हैं, तो ये चलाएं: 
+``sudo MMO_HTTPS=false apt install mattermost-omnibus``
+
+
+जब आपका सर्वर चल रहा हो , तो उसे डोमेन नाम (यानी mymattermostserver.com) के माध्यम से नेविगेट करें,जो आपके सर्वर (या यदि आप डोमेन नाम का उपयोग नहीं कर रहे हैं तो सर्वर के आईपी एड्रेस) को इंगित करता है। इसके बाद, एक टीम बनाएं और अपने मैटरमोस् सर्वर से जुड़ने के लिए लोगों को आमंत्रित करें।
+
+यदि आपको मैटरमोस्ट ओमनीबस स्थापित करने में कोई समस्या आ रही हो ,तो सामान्य त्रुटि संदेशों के लिए [ट्रबलशूटिंग गाइड](https://docs.mattermost.com/install/troubleshooting.html)  देखें, या समस्या निवारण सहायता के लिए [मैटरमोस्ट उपयोगकर्ता समुदाय](https://mattermost.com/ask-mattermost-community/) में शामिल हों।
+
+
+**उपयोगकर्ताओं को जोड़ें**
+
+अब अपनी टीम के लोगों को आमंत्रण भेजकर उन्हें जोड़ें:
+
+1. नेविगेशन पैनल के शीर्ष पर अपने यूजरनेम पर क्लिक करें। 
+2. Invite People पर क्लिक करें और उन लोगों के ईमेल पते एंटेर करें जिन्हें आप जोड़ना चाहते हैं।
+3. Invite Members पर क्लिक करें।
+
+जब आमंत्रित व्यक्ति लिंक के साथ ईमेल प्राप्त करता है, तो आपके सर्वर से जुड़ने के लिए यह एक सिंगल क्लिक होगा, जिस चैनल को आपने बनाया है, उसी में चैट करें या स्वयं के चैनल बनाएं। उपयोगकर्ताओं को जोड़ने के बारे में अधिक जानकारी [यहाँ](https://docs.mattermost.com/help/getting-started/managing-members.html#managing-members) उपलब्ध है।
+
+
+**अधिसूचना हब बनाए** 
+एक  ["heartbeat" चैनल](https://community.mattermost.com/core/channels/community-heartbeat) बनाएं और एक ही जगह  पर अपने  स्टेटस अपडेट और महत्वपूर्ण सूचनाएं भेजने के लिए अपने सबसे अधिक उपयोग किए जाने वाले ऐप्स और प्लग इन को मैटरमोस्ट के साथ एकीकृत करें।
+
+आप Main Menu > Plugin Marketplace के माध्यम से इंटीग्रेशन और प्लगइन्स पा और इंस्टॉल कर सकते हैं।
+
+
+GitHub, Jira और Jenkins से  आरंभ करें:
```

Comment:
Hmm .. Arambh kare doesn't really translate to "Get started with". It would mean "start with", which has a slightly different meaning. It's difficult though. Do you have a better choice of words here?